compiler/cpp/src/thrift/generate/t_swift_generator.cc

```diff
@@ -67,6 +68,11 @@ class t_swift_generator : public t_oop_generator {
         log_unexpected_ = true;
       } else if( iter->first.compare("async_clients") == 0) {
         async_clients_ = true;
+      } else if( iter->first.compare("async_servers") == 0) {
+        if (gen_cocoa_ == true) {
```

Comment:
Why can't we just write `if (gen_cocoa_ )`?

Reply:
Good point, thanks.